lib/logatron/logatron.rb

```diff
@@ -50,12 +50,28 @@ def debug(msg)
       logger.debug(msg)
     end
 
+    # Deprecated; use request_info/parse_request_info for transportation-layer neutral methods.
     def http_headers
       {
-        'X-Ascent-Log-Id' => msg_id
+        'X-Ascent-Log-Id' => msg_id,
+        'X-Ascent-Site' => site
       }
     end
 
+    def request_info
+      {
+        message_id: msg_id,
+        site: site
+      }
+    end
+
+    def parse_request_info(info)
```

Comment:
Also (and this is a nitpick) I would prefer to see `info ||= {}` rather than `if info`. The former is preferred since it "collapses" the branch, making the code easier to reason about. (This works unless calling `site=/msg_id=` has a side effect other than setting the attribute, which would be scary.)